### DISCUSSION TITLE
Crowdin - root folders

### DISCUSSION THREAD

**Discussion by davide-alghi** [YetiForceCompany/YetiForceCRM#15741] [General] [answered]
Hi,
what is the difference between the top folders on Crowdin?

"YetiForceCRM / developer"

**Comment by mariuszkrzaczkowski** [comment]
attach a screenshot, what folders are you talking about?

**Comment by davide-alghi** [comment, marked as answer]
![Schermata da 2022-02-23 08-20-25](https://user-images.githubusercontent.com/48508691/155276364-dba3d5ea-d147-4afd-a29a-0d7b2a98b838.png)

**Reply by mariuszkrzaczkowski** [reply]
fixed thanks for noticing it
